### PR TITLE
Added Method to create a CCMenuItemToggle from a CCArray with a target selector

### PR DESCRIPTION
**Pull request by CodeSnooker**
I have added a Method to create a CCMenuItemToggle from a CCArray with a target selector.

### CODE REVIEW

**Comment by shinriyo** [comment]
I hope the method. cocos2d-x for iPhone have the method

File: cocos2dx/menu_nodes/CCMenuItem.cpp [hunk 1]
```diff
@@ -714,6 +714,24 @@ CCArray* CCMenuItemToggle::getSubItems()
     return m_pSubItems;
 }
 
+CCMenuItemToggle * CCMenuItemToggle::createWithTarget(CCObject* target, SEL_MenuHandler selector, CCArray* menuItems)
+{
+    CCMenuItemToggle *pRet = new CCMenuItemToggle();
+    pRet->CCMenuItem::initWithTarget(target, selector);
+    pRet->m_pSubItems = CCArray::create();
+    pRet->m_pSubItems->retain();
```
Comment (minggo) [inline cocos2dx/menu_nodes/CCMenuItem.cpp:722]: Should not determine the return value of pRet and initWithTarget?

File: cocos2dx/menu_nodes/CCMenuItem.cpp [hunk 2]
```diff
@@ -714,6 +714,24 @@ CCArray* CCMenuItemToggle::getSubItems()
...
+    
+    for (int z=0; z < menuItems->count(); z++)
```
Comment (minggo) [inline cocos2dx/menu_nodes/CCMenuItem.cpp:724]: Should not determine the validation of menuItems?

File: cocos2dx/menu_nodes/CCMenuItem.cpp (hunk 1 above)
Comment (CodeSnooker) [inline cocos2dx/menu_nodes/CCMenuItem.cpp:722]: Sorry, I didn't get your statement.

File: cocos2dx/menu_nodes/CCMenuItem.cpp (hunk 2 above)
Comment (CodeSnooker) [inline cocos2dx/menu_nodes/CCMenuItem.cpp:724]: Validation can be added but are you sure that we have validation for cases as well while creating a CCMenuItemToggle object? 

I didn't find any.

Check the method body of:

```
bool CCMenuItemToggle::initWithTarget(CCObject* target, SEL_MenuHandler selector, CCMenuItem* item, va_list args)
```

and 

```
CCMenuItemToggle * CCMenuItemToggle::createWithTarget(CCObject* target, SEL_MenuHandler selector, CCMenuItem* item, ...)
```

**Comment by minggo** [comment]
You are right.
Thank you.

**Pull request merged**